src/modules/70-pipeline/strings/strings.en.yaml

```
@@ -239,15 +239,14 @@ jiraCreateStep:
   unsupportedRequiredFieldsError: "Changes can't be saved as unsupported fields are present - {{ fields }}"
   validations:
     summary: Summary is required
+  projectIssueTypeDisclaimer: Only permitted projects or issueTypes are being displayed. Please check if there are sufficient permissions in Jira to perform this action
```

nit: I think we use issue type everywhere
```suggestion
  projectIssueTypeDisclaimer: Only permitted projects or issue types are being displayed. Please check if there are sufficient permissions in Jira to perform this action.
```